pkg/context/keys.go

```diff
@@ -23,4 +23,6 @@ const (
 	JWTClaimsKey = "jwtClaims"
 	// WorkspaceKey is the context key for the workspace name in echo.Context
 	WorkspaceKey = "workspace"
+	// RequestRecievedTime is the context key for the starting time of a request made
+	RequestRecievedTime = "requestRecievedTime"
```

Comment:
I do the same type every single time when I write the word :sweat_smile: 
```suggestion
	// RequestReceivedTime is the context key for the starting time of a request made
	RequestReceivedTime = "requestReceivedTime"
```

Reply:
took me some time to see the difference 😅

Reply:
done in [631fdc6](https://github.com/codeready-toolchain/registration-service/pull/342/commits/631fdc67dfc3b7aadcf3233ef1ecff22d2455b14)